apps/namadillo/src/App/Transfer/TransferModule.tsx

```diff
@@ -144,6 +144,19 @@ export const TransferModule = ({
     return gasConfig ? getDisplayGasFee(gasConfig, chainAssetsMap) : undefined;
   }, [gasConfig]);
 
+  const availableAssets: AddressWithAssetAndAmountMap = useMemo(() => {
+    if (!source.availableAssets) return [];
+    return Object.keys(source.availableAssets).reduce(
+      (previous, current): AddressWithAssetAndAmountMap => {
+        if (source.availableAssets![current].amount.gt(0)) {
+          return { ...previous, [current]: source.availableAssets![current] };
+        }
+        return previous;
+      },
+      {}
+    );
+  }, [source.availableAssets]);
+
```

Comment:
This works great for me! Are we also going to figure the assets with zero balance displayed in the Overview?

Reply:
Hmmm I'll extract this filter into a new function and open a new PR for that! Thanks for remembering that